backend/app/api/routes/document_generation.py

```
@@ -0,0 +1,558 @@
+            "Access-Control-Allow-Headers": "Cache-Control"
+        }
+    )
+
+
+@router.post("/{project_id}/export", response_model=ExportResponse)
+async def create_project_export(
+    project_id: UUID,
+    request: ExportRequest,
+    background_tasks: BackgroundTasks,
+    export_service: ExportService = Depends(get_export_service),
+    user: User = Depends(get_current_user)
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)